docs-src/concepts/what-is-standard-visibility.md

```diff
@@ -1,10 +1,9 @@
 ---
 id: what-is-standard-visibility
-title: What is Standard Visibility?
+title: What is standard Visibility?
```

Comment:
why lowercase here?

Reply:
Because we're moving away from standard and advanced visibility as separate concepts (but we still support standard and advanced), this is the first step to remove it as a term from the glossary etc. 
I can revert back to cap-case but I wanted to formally deprecate it as a Temporal term.

Reply:
👍 there are other instances of "Standard V" if you wanted to find/replace now or at some later point

Reply:
Yes! Will try to catch them all in this PR.